### PR TITLE
fix: fix lack of copyUIDGID in swagger.yaml

### PR DESCRIPTION
**Pull request by ZYecho**
Signed-off-by: zhangyue <zy675793960@yeah.net>

**- What I did**
original fix at [#28923](https://github.com/moby/moby/pull/28923), but lack of bringing change to swagger.

**- How I did it**
add copyUIDGID param to swagger.yaml

**- How to verify it**
No need
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by olljanat** [comment]
@ZYecho CI build fails to
```
11:50:25 api/swagger.yaml
11:50:25   [2m623   error    syntax error: expected <block end>, but found '<block mapping start>'
```
because that yaml is not valid.

Also use your real name on sign off message.

**Comment by ZYecho** [comment]
> @ZYecho CI build fails to
> 
> ```
> 11:50:25 api/swagger.yaml
> 11:50:25   �[2m623   error    syntax error: expected <block end>, but found '<block mapping start>'
> ```
> 
> because that yaml is not valid.
> 
> Also use your real name on sign off message.

done.

**Comment by olljanat** [comment]
Derek add label: rebuild/windowsRS1

**Comment by thaJeztah** [comment]
Could you also add a note to the API history file? I see the original PR was merged for Docker 17.06, so the change should be included under the API v1.30 changes; https://github.com/moby/moby/blob/master/docs/api/version-history.md#v130-api-changes

File: api/swagger.yaml [hunk 1]
```diff
@@ -6229,6 +6229,10 @@ paths:
           in: "query"
           description: "If “1”, “true”, or “True” then it will be an error if unpacking the given content would cause an existing directory to be replaced with a non-directory and vice versa."
           type: "string"
+        - name: "copyUIDGID"
+          in: "query"
+          description: "If “1”, “true”, or “True” then it will copy UID/GID maps to the dest file"
```
Comment (thaJeztah) [inline api/swagger.yaml]: I see this is documented like this on other places as well, but perhaps we should just document `1` and `true`. 

Strictly speaking, any non-empty value should work; https://github.com/moby/moby/blob/4f0d95fa6ee7f865597c03b9e63702cdcb0f7067/api/server/httputils/form.go#L9-L13

But I don't think we should officially support all of that (so officially support just `1` and `true`)

**Review by thaJeztah — CHANGES_REQUESTED** [review]

**Comment by ZYecho** [comment]
@thaJeztah PTAL

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**